src/plugin_deno_loader.ts

```diff
@@ -263,9 +265,13 @@ export function denoLoaderPlugin(
         }
       });
 
+      const CSS_IMPORT_KIND = /^(import-rule|composes-from|url-token)$/;
       async function onResolve(
         args: esbuild.OnResolveArgs,
       ): Promise<esbuild.OnResolveResult | undefined> {
+        if (CSS_IMPORT_KIND.test(args.kind)) {
```

Comment:
This assertion indicates the import is coming from within a css file. The deno loader plugin shouldn't try resolving those paths as they are typically svgs or image files that the deno loader plugin can't handle.